plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -1547,6 +1548,42 @@ private void testReadV2Checkpoint(String resourceName)
         assertQuery("SELECT * FROM " + tableName, "VALUES (1, 2)");
     }
 
+    @Test
+    public void testTypeWideningNotSkippingUnsupportedPartitionColumns()
+            throws Exception
+    {
+        String tableName = "test_part_type_widening_" + randomNameSuffix();
+        Path tableLocation = Files.createTempFile(tableName, null);
```

Comment:
Please create tables under `catalogDir` so that we can remove the directory at the end of tests. Same for others. 
```suggestion
        Path tableLocation = catalogDir.resolve(tableName);
```